catalog/nsili/catalog-nsili-endpoint/src/main/java/com/connexta/alliance/nsili/endpoint/managers/OrderMgrImpl.java

```diff
@@ -63,20 +64,21 @@ public ValidationResults validate_order(OrderContents order, NameValue[] propert
     @Override
     public OrderRequest order(OrderContents order, NameValue[] properties)
             throws ProcessingFault, InvalidInputParameter, SystemFault {
-        OrderRequestImpl orderRequest = new OrderRequestImpl();
+        OrderRequestImpl orderRequestImpl = new OrderRequestImpl();
 
+        String id = UUID.randomUUID().toString();
         try {
-            _poa().activate_object_with_id("order".getBytes(Charset.forName(ENCODING)),
-                    orderRequest);
+            _poa().activate_object_with_id(id.getBytes(Charset.forName(ENCODING)),
+                    orderRequestImpl);
         } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-            LOGGER.warn("order : Unable to activate orderRequest object.");
+            LOGGER.warn("order : Unable to activate orderRequest object. {}", e.getLocalizedMessage());
```

Comment:
We can just throw e here, instead of getting the localized message.  Also recommend changing this to error level